gr-uhd/lib/CMakeLists.txt

```diff
@@ -77,6 +77,10 @@ add_library(gnuradio-uhd SHARED ${gr_uhd_sources})
 target_link_libraries(gnuradio-uhd ${uhd_libs})
 GR_LIBRARY_FOO(gnuradio-uhd RUNTIME_COMPONENT "uhd_runtime" DEVEL_COMPONENT "uhd_devel")
 
+if(UHD_VERSION VERSION_GREATER_EQUAL "3.9.0")
+  add_compile_definitions(UHD_USRP_MULTI_FILTER_API)
```

Comment:
Is there a difference between this and the existing #define here? At first glance, they appear to refer to the same feature.

https://github.com/EttusResearch/uhd/blob/release_003_009_000/host/include/uhd/usrp/multi_usrp.hpp#L33

Reply:
! I will take a look. I hadn't noticed those defines in UHD.

The PR has been stalled because `add_compile_definitions` isn't supported in our (current) minimum cmake version, but I think the current cmake PR actually bumps the min up to a version that does have it. If I can use this define then it's irrelevant.

Reply:
Actually I'd prefer using `target_compile_definitions(gnuradio-uhd PUBLIC UHD_USRP_MULTI_FILTER_API)`  which will transitively add that to the exported targets. (If public headers in UHD or gnuradio-uhd contain ifdef's based on that define).

Reply:
Cool to know about that option. I think the UHD define will work, but am test building my change  now with various UHD versions.

Reply:
Yes, probably the define in the UHD header is sufficient in this case (or even the right way).